packages/path-provider/lib/path_provider.dart

```diff
@@ -40,3 +40,16 @@ const _channel = const MethodChannel('plugins.flutter.io/path_provider');
       return null;
     return new Directory(path);
   }
+
+  /// Path to a directory where the application may access top level storage
+  ///
+  /// On iOS, This function returns null as it is not possible to access outside
```

Comment:
Sentences in comments are missing terminating `.`

Few typos in iOS sentence, change to: 
```
/// On iOS, this function returns null, as it is not possible to access the
/// file system outside the apps sandbox.
```